Sources/Aardvark/Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -21,13 +21,13 @@
 @import MessageUI;
 
 #import <CoreAardvark/AardvarkDefines.h>
+#import <CoreAardvark/ARKDefaultLogFormatter.h>
 #import <CoreAardvark/ARKLogMessage.h>
 #import <CoreAardvark/ARKLogStore.h>
 
 #import "ARKEmailBugReporter.h"
 #import "ARKEmailBugReporter_Testing.h"
 
-#import "ARKDefaultLogFormatter.h"
```

Comment:
I'm trusting that you've updated all imports.

Reply:
I think I got them all...